src/components/navigation/z-navigation-tabs/index.tsx

```diff
@@ -0,0 +1,127 @@
+import { Component, Prop, h, Listen, Element, State, Watch, Host } from '@stencil/core';
+
+import {
+  TabSizeBean,
+  TabSizeEnum,
+  TabOrientationBean,
+  TabOrientationEnum
+} from '../../../beans';
+import { ZNavigationTab } from '../z-navigation-tab';
+
+@Component({
```

Comment:
the components choildren state does not react to keyboard Tab input (only on mouse click)